media-libs/avbin-bin/avbin-bin-7.ebuild

```diff
@@ -15,13 +15,13 @@ SLOT="0"
 KEYWORDS="~amd64 ~x86 -*"
 IUSE=""
 
-src_install() {
-	if use amd64; then
-		cd "${WORKDIR}"/avbin-linux-x86-64-${PV} || die
-	elif use x86; then
-		cd "${WORKDIR}"/avbin-linux-x86-32-${PV} || die
-	fi
+if use amd64; then
+	S="${WORKDIR}"/avbin-linux-x86-64-${PV}
+elif use x86; then
+	S="${WORKDIR}"/avbin-linux-x86-32-${PV}
+fi
```

Comment:
That won't work as S is cached and the cache could also be generated on an hppa machine, which would result in S=. It needs to wrapped in some phases e.g. pkg_prepare()